apps/angular/pipe-easy/src/app/app.component.ts

```diff
@@ -1,21 +1,17 @@
 import { NgFor } from '@angular/common';
 import { Component } from '@angular/core';
+import { HeavyComputationPipe } from './shared/HeavyComputation.pipe';
 
 @Component({
   standalone: true,
-  imports: [NgFor],
   selector: 'app-root',
   template: `
     <div *ngFor="let person of persons; let index = index">
-      {{ heavyComputation(person, index) }}
+      {{ person | heavy: index }}
```

Comment:
very good, you could have used the new control flow syntax, but this is not the goal of this challenge.